client/scss/media.scss

```diff
@@ -10,29 +10,40 @@
 		&>a {
 			padding: 0px;
 			justify-content: center;
-			
+
 			h1 {
 				width: auto;
+				font-size: 12vw;
```

Comment:
I don't understand this comment.  Scale down the font size of the stylized site title to make it fit the width.